commands/upgrade.ts

```diff
@@ -0,0 +1,107 @@
+import { readFile } from 'node:fs/promises'
+import { join, resolve } from 'node:path'
+import type { Options } from 'jscodeshift'
+import { run as jscodeshift } from 'jscodeshift/src/Runner'
+import prompts from 'prompts'
+import { coerce, compare } from 'semver'
+import { TRANSFORM_OPTIONS } from '../config'
+import { onCancel, promptSource } from '../utils/share'
+
+const transformerDirectory = join(__dirname, '../', 'transforms')
+
+export async function upgrade(source?: string) {
+  const sourceSelected = source || (await promptSource('Which directory should the codemods be applied to?'))
+
+  if (!sourceSelected) {
+    console.info('> Source path for project is not selected. Exits the program. \n')
+    process.exit(1)
+  }
+
+  try {
+    const packageJsonPath = resolve(sourceSelected || '', 'package.json')
+    const packageJson = JSON.parse(await readFile(packageJsonPath, 'utf8'))
+
+    const codemods = suggestCodemods(packageJson)
+
+    if (codemods.length === 0) {
+      console.info('> No codemods are suggested for this project. \n')
+
+      return
+    }
+
+    const { codemodsSelected } = await prompts(
+      {
+        type: 'multiselect',
+        name: 'codemodsSelected',
+        message: `The following 'codemods' are recommended for your upgrade. Select the ones to apply.`,
```

Comment:
we recommend them only by checking version in the `package.json` file. It doesn't necessarily bring value. Let's just check if express is there and ask user to select codemods.

Reply:
But what if there are codemods for version 6 and we're on version 5? We wouldn't want to recommend codemods from version 4, right?